### DISCUSSION TITLE
X想问问之前的新手指导还有吗？我使用的是zotero6，下载的也是6的最新版，但找不到之前对一些功能的解释了

### DISCUSSION THREAD

**Discussion by 1584136307qq** [windingwind/zotero-better-notes#709] [Q&A] [answered]
想找这个快速引注的使用方法
/
![6HZ5_I7MXFAF4_4O1VYF9IP](https://github.com/windingwind/zotero-better-notes/assets/145474511/db528482-88c7-4436-8a80-fe759b6af094)

**Comment by 1584136307qq** [comment, marked as answer]
我学会了。
①搜索”标题/作者“后空格，等待搜索的文件出现
<img width="50" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/145474511/b1ea113c-4a2e-45f6-a60f-229e1c1ea15e">
（注意看有空格）
②点击需要的文件，在用”enter"键
<img width="212" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/145474511/a1900f51-0fa3-4aec-8a78-dfd694d71e27">
<img width="234" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/145474511/adb83ed9-2568-42b9-a878-9b4b6e048210">
③就可以了

**Reply by guolizhang-tyler** [reply]
嗯嗯 谢谢
